internal/install/go_task_recipe_executor.go

```diff
@@ -116,6 +120,20 @@ func setVarsFromSystemInfo(t *taskfile.Taskfile, m discoveryManifest) {
 	t.Vars.Merge(&v)
 }
 
+func setLiveVars(t *taskfile.Taskfile, vars map[string]interface{}) error {
```

Comment:
Should we change this to `setStaticVars`?  `setProgrammaticVars`? Naming things is hard.

Reply:
Good call.